src/Jobs/BasePackageJob.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Flagrow\Bazaar\Jobs;
+
+use Carbon\Carbon;
+use Flagrow\Bazaar\Composer\ComposerCommand;
+use Flagrow\Bazaar\Composer\ComposerEnvironment;
+use Flagrow\Bazaar\Composer\ComposerOutput;
+use Flagrow\Bazaar\Task;
+//use Illuminate\Bus\Queueable;
+//use Illuminate\Queue\SerializesModels;
+//use Illuminate\Queue\InteractsWithQueue;
+//use Illuminate\Contracts\Queue\ShouldQueue;
```

Comment:
I'm not sure how to handle that. I could require `illuminate/queue` and use the Sync driver there. That's another dependency but that way we can use the queuable contract. 

Or I just remove them and leave the jobs as simple classes with a `handle` method